### PR TITLE
Log potentially invalid responses for better issue understanding

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
This should help to solve issues like https://github.com/appium/appium/issues/8376 by logging the actual proxy response.

### CODE REVIEW

File: lib/no-session-proxy.js [hunk 1]
```diff
@@ -26,6 +28,9 @@ class NoSessionProxy extends JWProxy {
     res.headers = response.headers;
     res.set('Content-type', response.headers['content-type']);
     body = util.safeJsonParse(body);
+    if (_.isString(body)) {
+      log.warn(`The '${_.trunc(body, 300)}' response is not valid JSON object`);
```
Comment (imurchie) [inline lib/no-session-proxy.js]: `_.trunc` no longer exists. It's `_.truncate(body, {length: 300})` now.

**Comment by mykola-mokhnach** [comment]
yep, it will be easy to create an unit test for this method if think it could be useful

**Comment by imurchie** [comment]
That would be good. This change won't break anything, I know that, but it's hard to know that it solves the problem. :)

**Comment by mykola-mokhnach** [comment]
I have figured out that the exception is thrown not in the place where I expected that. The investigation will eventually take more time.

**Comment by imurchie** [comment]
That was my fear. :(

**Pull request closed without merging**